### ISSUE TITLE
How to best reexport greta internals

### ISSUE THREAD

**Issue by njtierney** [greta-dev/greta.distributions#12]
Currently the internals are internally exported here:

https://github.com/greta-dev/greta.distributions/blob/main/R/internals.R#L1-L18

However I think if we want users to use this package to create their own greta distributions, then we need to make these exported?

In which case, we need to add documentation, so it might look like this?

```r
#' @param distribution
#' @param ...
#' @export
distrib <- greta::.internals$nodes$constructors$distrib
```

It feels a bit funny to be reexporting internal functions and then giving them documentation here?

**Comment by njtierney** [comment]
We also need all of the helper functions from https://github.com/greta-dev/greta/blob/master/tests/testthat/helpers.R to establish tests, at the moment we could just copy these over into `tests/testthat/helpers.R` here - are you happy with that approach, @goldingn ?

**Comment by goldingn** [comment]
Yep, I think copying them over is a good stopgap. We can think if there's a tidier option (like a standalone package of testing functions) later.
